doc/modules/sgd.rst

```diff
@@ -512,15 +512,15 @@ For regression the default learning rate schedule is inverse scaling
     \eta^{(t)} = \frac{eta_0}{t^{power\_t}}
 
 where :math:`eta_0` and :math:`power\_t` are hyperparameters chosen by the
-user via ``eta0`` and ``power_t``, resp.
+user via ``eta0`` and ``power_t``, respectively.
 
 For a constant learning rate use ``learning_rate='constant'`` and use ``eta0``
 to specify the learning rate.
 
 For an adaptively decreasing learning rate, use ``learning_rate='adaptive'``
 and use ``eta0`` to specify the starting learning rate. When the stopping
 criterion is reached, the learning rate is divided by 5, and the algorithm
-does not stop. The algorithm stops when the learning rate goes below 1e-6.
+does not stop. The algorithm stops when the learning rate goes below :math:`1 \times e^-6`.
```

Comment:
Similarly here, `1e-6` refers to the parameter's computational value, so it's more of a programmatic expression than a mathematical expression
```suggestion
does not stop. The algorithm stops when the learning rate goes below `1e-6`.
```